src/stylesheets/components/_accordion.scss

```diff
@@ -144,3 +144,17 @@
     }
   }
 }
+
+.sds-card {
+  .usa-accordion--bordered.sds-accordion-next {
+    @include u-padding-x(0);
+  }
+  .sds-accordion-next {
+    .sds-accordion-title {
+      @include u-text('secondary-dark');
+    }
+  }
+  .sds-accordion-next .mat-accordion .mat-expansion-panel .mat-expanded .sds-accordion-title {
```

Comment:
Is the angular material's css - `.mat-accordion .mat-expansion-panel .mat-expanded` needed here? If `.sds-accordion-next..sds-accordion-title` also applies the style, then that would be better because then we don't have to assume angular material is being used.